site/assets/scss/docs.scss

```diff
@@ -51,3 +51,9 @@
 @import "syntax";
 @import "anchor";
 @import "algolia";
+
+// Temp
+
+.offcanvas{
+  z-index: 1072 // over .bd-subnavbar
```

Comment:
We should probably move this in the docs variables and reuse it.

Reply:
Or as override on `site/assets/scss/docs.scss` before  the general variables import, but css-linter is complaining.

Reply:
@MartijnCuppens can you please have a look when you have some time?

I also added another TODO regarding the `h4`s

Reply:
Why is this added? The z-index of the off canvas component is higher than `.bd-subnavbar` anyway?

Reply:
it was added because  in a previous version, .bd-subnavbar had 1071 z-index and was covering offcanvas. I can make a check to find if it is changed. 

(offcanvas is on hold about 10 months, so many things have changed and sure  I have missed some of them)

Reply:
Finally, it was redundant